libraries/types/proposal_period_levels_map/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(HEADERS include/dag/proposal_period_levels_map.hpp)
```

Comment:
as I have already commented I do not think this should be separate lib

Reply:
so why and how was this resolved when it is still here ?

Reply:
Because there is another comment about this issue. I think there is no need to leave few unresolved conversations on 1 item